using-mobile-extensions/mobile-core/mobile-core-api-reference.md

```diff
@@ -799,6 +799,127 @@ ACPCore.setAppGroup("app-group-id")
 {% endtab %}
 {% endtabs %}
 
+## Public Classes
+
+{% tabs %}
+{% tab title="Android" %}
+
+#### Android
+
+##### AdobeCallback
+
+This class provides the interface to receive results when the async APIs perform the requested action.
+
+```java
+public interface AdobeCallback<T> {    
+    void call(final T value);
+}
+```
+
+##### AdobeCallbackWithError
+
+This class provides the interface to receive results or an error when the async APIs perform the requested action. When using this class, if the request cannot be completed within 500ms or an unexpected error occurs, the request is aborted and the _fail_ method is called with the corresponding _AdobeError_.
```

Comment:
500ms is the Identity timeout but it can e different for each extension, can we say something like `if the request cannot be completed within the default timeout or an unexpected..`?